planemo/commands/cmd_shed_upload.py

```diff
@@ -102,8 +103,33 @@ def __handle_upload(ctx, path, **kwds):
     message = kwds.get("message", None)
     if message:
         update_kwds["commit_message"] = message
+
+    repo_id = __find_repository(ctx, tsi, path, kwds)
+    if repo_id is None:
+        return -1
+
+    try:
+        tsi.repositories.update_repository(repo_id, tar_path, **update_kwds)
+    except Exception as e:
+        exception_content = e.read()
+        try:
+            upstream_error = json.loads(exception_content)
+            error(upstream_error['err_msg'])
+        except:
```

Comment:
Bare `except`s are an anti-pattern - https://docs.python.org/2/howto/doanddont.html#except - can we modify this to be `except Exception`. I would be happy to make the change post-merge.